src/consts.js

```diff
@@ -56,3 +56,7 @@ export const cancelFeeClaimReturned = {
 };
 
 export const MAX_RECORDS = '1000';
+
+export const DCB_USER = {
```

Comment:
Could you please clarify this user will be created to BE and always have the same ID?

Reply:
Basically, these are the virtual patrons that are created by BE scripts only.
There can be many users with lastName "DcbSystem" and type "dcb".
Here we need check for user type as it is not obtained in the checkin response.